splunk_eventgen/lib/plugins/output/metric_httpevent.py

```diff
@@ -0,0 +1,295 @@
+from __future__ import division
+
+import logging
+import random
+import urllib
+
+from outputplugin import OutputPlugin
```

Comment:
Why don't we bust out the "core" of httpevent into a object, and then set both plugins to inherit from the core?

Reply:
Agree. A lot of duplicate code.